<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#loginButton
{
	font: 25 10pt &quot;Microsoft YaHei&quot;;
	border-image: url(:/Resources/images/Button.jpg);
    border-radius:5px;
}

QPushButton#loginButton:hover
{
    border-image: url(:/Resources/images/Button.jpg);
}

QPushButton#loginButton:pressed
{
    border-image: url(:/Resources/images/Button.jpg);
    padding-left:3px;
    padding-top:3px;
}
</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#exitButton
{
	font: 25 10pt &quot;Microsoft YaHei&quot;;
    border-image: url(:/Resources/images/Button.jpg);
    border-radius:5px;
}

QPushButton#exitButton:pressed
{
    border-image: url(:/Resources/images/Button.jpg);
    padding-left:3px;
    padding-top:3px;
}
</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>9</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#applyButton
{
	font: 25 9pt &quot;Microsoft YaHei&quot;;
    border-image: url(:/Resources/images/Button.jpg);
    border-radius:5px;
}

QPushButton#applyButton:pressed
{
    border-image: url(:/Resources/images/Button.jpg);
    padding-left:3px;
    padding-top:3px;
}
</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>51</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>51</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>6</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>61</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>帐号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newAccountEdit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>120</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswordEdit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>170</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>6</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>51</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>290</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#submitButton
{
	font: 25 10pt &quot;Microsoft YaHei&quot;;
    border-image: url(:/Resources/images/Button.jpg);
    border-radius:5px;
}

QPushButton#submitButton:pressed
{
    border-image: url(:/Resources/images/Button.jpg);
    padding-left:3px;
    padding-top:3px;
}
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>230</y>
     <width>101</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="checkPasswordEdit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>240</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>6</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>290</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>10</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#cancelButton
{
	font: 25 10pt &quot;Microsoft YaHei&quot;;
    border-image: url(:/Resources/images/Button.jpg);
    border-radius:5px;
}

QPushButton#cancelButton:pressed
{
    border-image: url(:/Resources/images/Button.jpg);
    padding-left:3px;
    padding-top:3px;
}
</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>400</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="close1Button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#close1Button
{
    border-image: url(:/Resources/images/close.png);
}
QPushButton#close1Button:hover
{
	border-image: url(:/Resources/images/close_hover.png);
}
QPushButton#close1Button:pressed
{
    border-image: url(:/Resources/images/close_press.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="min1Button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#min1Button
{
    border-image: url(:/Resources/images/min.png);
}
QPushButton#min1Button:hover
{
	border-image: url(:/Resources/images/min_hover.bmp);
}
QPushButton#min1Button:pressed
{
    border-image: url(:/Resources/images/min_press.bmp);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="close2Button">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#close2Button
{
    border-image: url(:/Resources/images/close.png);
}
QPushButton#close2Button:hover
{
	border-image: url(:/Resources/images/close_hover.png);
}
QPushButton#close2Button:pressed
{
    border-image: url(:/Resources/images/close_press.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="min2Button">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#min2Button
{
    border-image: url(:/Resources/images/min.png);
}
QPushButton#min2Button:hover
{
	border-image: url(:/Resources/images/min_hover.bmp);
}
QPushButton#min2Button:pressed
{
    border-image: url(:/Resources/images/min_press.bmp);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resources/images/block.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>loginButton</zorder>
  <zorder>exitButton</zorder>
  <zorder>applyButton</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>accountEdit</zorder>
  <zorder>passwordEdit</zorder>
  <zorder>label_3</zorder>
  <zorder>newAccountEdit</zorder>
  <zorder>newPasswordEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>submitButton</zorder>
  <zorder>label_5</zorder>
  <zorder>checkPasswordEdit</zorder>
  <zorder>cancelButton</zorder>
  <zorder>close1Button</zorder>
  <zorder>min1Button</zorder>
  <zorder>close2Button</zorder>
  <zorder>min2Button</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
